test/unit/shebang/output.js

```diff
@@ -0,0 +1,95 @@
+module.exports =
```

Comment:
Should the shebang be copied to the output entrypoint file?

Reply:
And also possibly setting executable permissons (or copy the permissons from the entrypoint)?